src/getCommands.js

```diff
@@ -8,8 +9,14 @@ const findPlugins = require('./findPlugins');
  * @param  {String}       name Name of the plugin
  * @return {Array|Object}      Plugin's commands
  */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
+const getPluginCommands = cwd => name => {
+  const nested = path.join(cwd, 'node_modules', name);
+
+  /* For rnpm installed locally with flat dependencies structure */
+  const flat = path.join(cwd, '..', name);
```

Comment:
Yeah, you are right, both solutions are good. But if you will feel like the array implementation works as well, give it a go!